sv_pipeline/load_data.py

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env python
+
+import argparse
+from datetime import datetime
+from elasticsearch import helpers as es_helpers
+import re
+import subprocess
+from tqdm import tqdm
+
+from hail_scripts.shared.elasticsearch_client import ElasticsearchClient
+from hail_scripts.shared.elasticsearch_utils import ELASTICSEARCH_INDEX
+
+GS_SAMPLE_PATH = 'gs://seqr-datasets/v02/GRCh38/RDG_WES_Broad_Internal/base/projects/{project_guid}/{project_guid}_{file_ext}'
```

Comment:
I'm not entirely sure on what this doe does or why its useful here. Can you explain a bit more?

Reply:
I like using a resources file since it makes the code a little easier to read, and I like that I only have to update a path once if it's a resource that I use often/in multiple scripts